openstack_ansibleee/settings

```diff
@@ -1,6 +1,6 @@
 # If no output is detected from ansible in this number of seconds the execution will
 # be terminated.
-idle_timeout: ${RUNNER_IDLE_TIMEOUT:-600}
```

Comment:
I think this is the ansible runner default https://ansible.readthedocs.io/projects/runner/en/stable/intro/#env-settings-settings-for-runner-itself. Not sure if we should reduce it.

We probably could find the better way to know that the job has failed?

Reply:
I tried to find something in the ansible-runner to understand why it hangs when it fails, but I didn't find anything